<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Search</class>
 <widget class="QDialog" name="Search">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="searchLineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>431</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Wyszukiwana Fraza:</string>
   </property>
  </widget>
  <widget class="QTableView" name="searchingTableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>551</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="filtrGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>551</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Filtruj wg.:</string>
   </property>
   <widget class="QRadioButton" name="allRadioButton_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Wszystkie</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="ingrRadioButton_4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Składniki</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="recRadioButton_5">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Przepisy</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="sortGroupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>551</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Sortuj wg.:</string>
   </property>
   <widget class="QRadioButton" name="alphRadioButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Alfabetycznie rosnąco</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="alphRevRadioButton_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Alfabetycznie malejąco</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="defaultRadioButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Domyślne</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>60</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Szukaj</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
